scripts/variantstore/beta_docs/gvs-outputs.md

```diff
@@ -0,0 +1,126 @@
+# Working with the outputs of the Genomic Variant Store
+
+## Applying the GVS Joint Calling Filters
+During joint calling using GVS, filters are tagged onto the data based on the filtering model. However, the VCFs we output from GVS are not hard filtered, which means we include the filters in the FT flag and have a recommended filtering approach. Here, we outline that approach.
+
+Unfiltered variants will have “.” or PASS in the site level filters fields in the VCFs. Filtered variants will have the filter name in the site level filters of the VCF (FILTER). We recommend that researchers do not include variant sites that were filtered in their analyses. Additionally, GATK VETS filters are allele-specific and we recommend filtering out genotypes that have the FT tags outlined below.
+
+| Site-level Filters and Allele Specific Filters | Definition                                                                                                                    | Details                                                                                                                                                                                                                                                                                                                                               |
+|------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| NO_HQ_GENOTYPES                                | Site has no high quality variant genotypes                                                                                    | No high-quality genotype (GQ>=20, DP>=10, and AB>=0.2 for heterozygotes) called for the variant. Allele Balance (AB) is calculated for each heterozygous variant as the number of bases supporting the least-represented allele over the total number of base observations.  In other words, min(AD)/DP for diploid GTs.                              |
```

Comment:
but we have passing genotypes with GQ 0...

Reply:
It's very possible these definitions are out of date, I pulled them from a document from when we used VQSR. So appreciate more folks weighing in on how we define this filter.

Reply:
Got it, thank you for that clarification. I'll try to add that info to the documentation as well!